IPython/nbconvert/utils/lexers.py

```diff
@@ -49,10 +49,14 @@
 line_re = re.compile('.*?\n')
 
 ipython_tokens = [
-  (r'(\%+)(\w+)\s+(\.*)(\n)', bygroups(Operator, Keyword,
+  (r'(%)(sx|sc|system)(.*)(\n)', bygroups(Operator, Keyword,
                                        using(BashLexer), Text)),
-  (r'(\%+)(\w+)\b', bygroups(Operator, Keyword)),
```

Comment:
I think cell magics will require a separate rule because they should extend the match to the end of the cell and not terminate on `\n`.  I'll add some test cases for that.